exercises/bowling/bowling_test.py

```diff
@@ -0,0 +1,261 @@
+import unittest
+
+from example import BowlingGame
+
+
```

Comment:
Could you please also leave a comment stating what version of `canonical-data.json` the tests were adopted as discussed in #784 if aplicable?
The format is:
```text
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```